README.md

```diff
@@ -594,6 +594,11 @@ css(styles2.globals);
 
 It isn't determinate whether divs will be red or blue.
 
+## Minify class names
+
+Minify class names by setting the environment variable `process.env.APHRODITE_KEYS` 
```

Comment:
I'm a little hesitant to require that folks use a new env variable in order to benefit from this.